dotCMS/src/main/java/com/dotcms/rest/api/v1/categories/CategoriesResource.java

```diff
@@ -119,20 +128,22 @@ public final Response getCategories(@Context final HttpServletRequest httpReques
             @Context final HttpServletResponse httpResponse,
```

Comment:
Let's add some Javadoc here, and a Postman test taking the latest changes into account.

Reply:
Javadoc added - please review again. I have also added new postman tests for the changes I have done in the listing API

Reply:
Javadoc added